src/NuGet.Clients/NuGet.PackageManagement.UI/Xamls/LicenseAcceptanceWindow.xaml

```diff
@@ -99,82 +98,73 @@
       </DataTemplate>
     </ResourceDictionary>
   </Window.Resources>
-  <Grid>
-    <Grid.RowDefinitions>
-      <RowDefinition
+  <ScrollViewer x:Name="sv" CanContentScroll="False" VerticalScrollBarVisibility="Auto" HorizontalScrollBarVisibility="Auto">
```

Comment:
Current configuration won't enable this scrollbar, but it's a simple way to find Viewport dimensions, for now.